Source/Xamarin/Prism.Forms/Navigation/INavigationService.cs

```diff
@@ -39,7 +38,7 @@ public interface INavigationService
         /// <param name="parameters">The navigation parameters</param>
         /// <remarks>Navigation parameters can be provided in the Uri and by using the <paramref name="parameters"/>.</remarks>
         /// <example>
-        /// Navigate(new Uri("MainPage?id=3&name=brian", UriKind.RelativeSource), parameters);
+        /// Navigate(new Uri("MainPage?id=3&amp;name=brian", UriKind.RelativeSource), parameters);
```

Comment:
Should be "NavigateAsync"

Reply:
Interesting.  I'll modify.  That is the code that came from the Prism repo.  I only modifed the &amp;

I'll modify it